### PR TITLE
Toast comp

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Desc: Added Snackbar Component and added the animation for fadeIn and fadeOut

### CODE REVIEW

File: docs/docs.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const snackbar = document.getElementById("snackbar");
+
+const showSnackbar = () => {
+    snackbar.className = "show";
+    setTimeout(() => { snackbar.className = snackbar.className.replace("show", ""); }, 3000);
```
Comment (t007rushi) [inline docs/docs.js:5]: Liked clever use

File: utils/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    padding: 1rem;
+    position: fixed;
+    z-index: 1;
+    left: 45%;
+    bottom: 1rem;
+}
+
+#snackbar.show {
+    visibility: visible;
+    animation: fadeIn 0.5s, fadeOut 0.5s 3s;
```
Comment (t007rushi) [inline utils/snackbar.css:16]: Liked this pop up of toast

**Review by t007rushi — APPROVED** [review]
Just add ```Toast Navigation``` in NavBar

File: utils/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#snackbar {
+    visibility: hidden;
+    max-width: 18rem;
+    min-width: 10rem;
+    background-color: var(--primary);
+    color: var(--white);
+    text-align: center;
```
Comment (paarthbajaj) [inline utils/snackbar.css]: You already have a utility class for text-align center

**Review by paarthbajaj — APPROVED** [review]
Looks clean. You can merge just have a look on my comment and try to use your own made utility classes.

**Pull request merged**